docs/HOWTO-hi.md

```diff
@@ -10,18 +10,18 @@
 
 हम नए योगदानकर्ताओं का स्वागत करते हैं; यहां तक ​​कि उन लोगों के लिए जो GitHub पर अपना पहला Pull Request (PR) करते हैं। यदि आप उनमें से एक हैं, तो यहां कुछ संसाधन हैं जो मदद कर सकते हैं:
 
-* [About Pull Requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) *(in english)*
-* [Creating a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) *(in english)*
-* [GitHub Hello World](https://docs.github.com/en/get-started/quickstart/hello-world) *(in english)*
-* [YouTube - GitHub Tutorial For Beginners](https://www.youtube.com/watch?v=0fKg7e37bQE) *(in english)*
-* [YouTube - How To Fork A GitHub Repo and Submit A Pull Request](https://www.youtube.com/watch?v=G1I3HF4YWEw) *(in english)*
-* [YouTube - Markdown Crash Course](https://www.youtube.com/watch?v=HUBNt18RFbo) *(in english)*
+* [पुल अनुरोध के बारे में](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) *(Pull Request)*
```

Comment:
prefer to leave the annotation as *(in english)* The rest look good!

Reply:
> prefer to leave the annotation as _(in english)_ The rest look good!

I'm soo sorry Sir that i didn't saw your message..... I'm correcting it right now sir.

Reply:
Solved at 4ee30c7d74ea6b4f5bdf248932fe56384418f12f